doc/dev/doc/linq.md

```diff
@@ -0,0 +1,49 @@
+﻿# LINQ Provider (BETA)
+Hazelcast .Net Client provides a LINQ provider over `IHMap`. Provider is currently in BETA version.
```

Comment:
Can you use a "> NOTE [WARNING]" syntax (that's not the syntax, grep for it in the doc) which will create a big blue or orange block with the "beta" text in it, so it's highly visible? and maybe a small sentence expaining ppl may see issues or there may be changes in the code later on?

Reply:
Fixed at https://github.com/hazelcast/hazelcast-csharp-client/pull/759/commits/eedb97b777a60525334a90a76da72e794066f235.

Reply:
OK ... what I meant was, you'll need to find the syntax in another file 'cos I don't remember it :) I've searched it for you, for instance in `doc/dev/doc/cpsubsystem.md`:
```

> [!WARNING]
> LINQ support is currently in BETA version. There may be breaking changes on further releases.

```

Reply:
Fixed at https://github.com/hazelcast/hazelcast-csharp-client/pull/759/commits/f77d3946bf50eb5ae9af1fcb3083b55d73d848a3.